components/director/pkg/graphql/app_validation.go

```diff
@@ -1,19 +1,42 @@
 package graphql
 
 import (
+	"os"
+	"strconv"
+
 	validation "github.com/go-ozzo/ozzo-validation"
 	"github.com/kyma-incubator/compass/components/director/pkg/inputvalidation"
 )
 
+const appNormalizationEnvVar = "APP_NAME_NORMALIZATION_ENABLED"
+
+var appNameNormalizationEnabled bool
+
+func init() {
+	appNameNormalizationEnabled = false
+
+	value, isSet := os.LookupEnv(appNormalizationEnvVar)
+	if isSet {
+		parsedValue, err := strconv.ParseBool(value)
+		if err == nil {
+			appNameNormalizationEnabled = parsedValue
+		}
+	}
+}
+
 func (i ApplicationRegisterInput) Validate() error {
```

Comment:
Should we also add a test case in `app_validation_test.go`?

Reply:
Not possible due to the fact that app_validation.go has `init()` method which is called before any tests so we cannot plug the env variable with the right value.